### PR TITLE
{math,numlib}[intel/2015b] CGAL v4.7, MPFR v3.1.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8087173c6afb34045cda for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/c/CGAL/CGAL-4.7-intel-2015b-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+dependencies = [
+    ('Boost', '1.59.0', versionsuffix),
+    ('MPFR', '3.1.3', '-GMP-6.1.0'),
+    ('Qt5', '5.5.1', '-Mesa-%s' % mesaver),
+    ('Mesa', '11.0.8', versionsuffix),
+    ('libGLU', '9.0.0', '-Mesa-%s' % mesaver),
+]
+
+configopts = "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.so -DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.so "
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/CGAL/CGAL-4.7-intel-2015b-Python-2.7.11.eb]: `SHEXT`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5ff4f6c27584f62d407c for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**